Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM java:openjdk-7-jre
+FROM java:openjdk-8-jre
 MAINTAINER spiddy <d.kapanidis@gmail.com>
 
 RUN echo "deb http://downloads.sourceforge.net/project/sonar-pkg/deb binary/" >> /etc/apt/sources.list
-RUN apt-get update && apt-get clean ### Sonar version 5.1 - timestamp
+RUN apt-get update && apt-get clean ### Sonar version 5.6 - timestamp
 
-RUN apt-get install -y --force-yes sonar=5.1
+RUN apt-get install -y --force-yes sonar
```

Comment:
Can you add specific version of sonar here? This will avoid rebuild to drift to newest versions without supervision.